### PR TITLE
Assignment7+8 @ZackWard-student

### PR DESCRIPTION
**Pull request by jmplett**
@ZackWard-student

### CODE REVIEW

**Comment by mpennock5** [comment]
It looks like you guys pretty much got everything down. I couldn't find anything part of the requirements missing. I noticed that only the two part jokes are supported but nothing specified that we had to support the single part jokes. The menu appears a little small compared to banner that its on but that's just nitpicking honestly. Good job!!

File: Web/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                <a href="https://v2.jokeapi.dev/joke/Programming "><button>Joke API</button></a>
+            </div>
+        </div>
+
+        <script src="js/index.js "></script>
+        <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js "></script>
+        <script src="https://kit.fontawesome.com/8808ce2e4c.js " crossorigin="anonymous "></script>
+    </body>
+
+</html>
```
Comment (mpennock5) [inline Web/index.html:49]: It looks like you guys pretty much got everything down. I couldn't find anything part of the requirements missing. I noticed that only the two part jokes are supported but nothing specified that we had to support the single part jokes. The menu appears a little small compared to banner that its on but that's just nitpicking honestly. Good job!!

**Comment by TylerRose8** [comment]
Reviewing

**Review by mmwoodfo — COMMENTED** [review]
This looks great - good job

**Pull request closed without merging**